apps/crn-server/src/data-providers/contentful/research-tag.data-provider.ts

```diff
@@ -120,3 +125,18 @@ const parseResearchTag = (item: ResearchTagItem): ResearchTagDataObject => {
       : undefined,
   };
 };
+
+export const parseResearchTags = (
```

Comment:
It's the parse function that comes from the user data provider, I extracted it to avoid copy pasting to all data providers. I could use the parseResearchTag function but that would mean querying data that we don't actually use for tag display so it seems wasteful.